ts/pages/staking/wizard/remove.tsx

```diff
@@ -0,0 +1,339 @@
+
+export interface StakingWizardProps {}
+
+enum Steps {
+    Initial,
+    Confirm,
+    WaitingForRemoval,
+    WaitingForConfirmation,
+    Done,
+}
+
+const Container = styled.div`
+    max-width: 1390px;
+    margin: 0 auto;
+    position: relative;
+`;
+
+const CenteredHeader = styled.h2`
+    max-width: 440px;
+    margin: 0 auto;
+    font-size: 34px;
+    line-height: 1.23;
+    text-align: center;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 175px;
+`;
+
+const ButtonWithIcon = styled(Button)`
+    display: flex;
+    width: 100%;
+    justify-content: center;
+    align-items: center;
+`;
+
+const SpinnerContainer = styled.span`
+    display: inline-block;
+    margin-right: 10px;
+`;
+
+const NumberInputContainer = styled.div`
+    margin-bottom: 60px;
+`;
+
+const Message = styled.div`
+    text-align: center;
+    margin-top: 30px;
+    margin-bottom: 130px;
+
+    h2 {
+        font-size: 34px;
+        line-height: 1.24;
+        width: 100%;
+        margin-bottom: 15px;
+    }
+
+    p {
+        font-size: 18px;
+        line-height: 1.45;
+        color: ${colors.textDarkSecondary};
+        font-weight: 300;
+        max-width: 340px;
+        margin: 0 auto;
+    }
+`;
+
+const ButtonsContainer = styled.div`
+    display: flex;
+
+    & > button {
+        flex: 1;
+        margin-right: 30px;
+
+        &:last-child {
+            margin-right: 0;
+        }
+    }
+`;
+
+const IntroHeader = styled.h1`
+    font-size: 36px;
+    font-weight: 300;
+    line-height: 1.1;
+    margin-bottom: 15px;
+
+    @media (min-width: 768px) {
+        font-size: 50px;
+    }
+`;
+
+const IntroDescription = styled.h2`
+    font-size: 17px;
+    font-weight: 300;
+    color: ${colors.textDarkSecondary};
+    line-height: 1.44;
+    margin-bottom: 30px;
+    max-width: 340px;
+
+    @media (min-width: 768px) {
+        margin-bottom: 60px;
+        font-size: 18px;
+    }
+`;
+
+interface Data {
+    currency: string;
+    amountStaked: BigNumber;
+}
+
+const getData = async (): Promise<Data> => {
+    return new Promise<Data>((resolve, _reject) => {
+        resolve({
+            currency: 'ZRX',
+            amountStaked: new BigNumber(1000000),
+        });
+    });
+};
+
+export const RemoveStake: React.FC<StakingWizardProps> = props => {
+    const [step, setStep] = React.useState<Steps>(Steps.Initial);
+    const [value, setValue] = React.useState<null | string>(null);
+    const [parsedValue, setParsedValue] = React.useState<null | number>(null);
+    const [isError, setIsError] = React.useState<boolean>(false);
+    const [data, setData] = React.useState<Data>({
+        currency: 'ZRX',
+        amountStaked: new BigNumber(0),
+    });
+
+    const isStepWaiting = step === Steps.Confirm ||
+        step === Steps.WaitingForConfirmation ||
+        step === Steps.WaitingForRemoval;
+
+    const onValueChange = (newValue: string): void => {
+        const parsed = parseInt(newValue, 0);
```

Comment:
shouldn't radix 10? or do we want to support other bases?

Reply:
In case we don’t want to supper other bases (which is probably the case): 

https://github.com/0xProject/website/pull/39